lib/Migration/Version0Date20240826193657.php

```diff
@@ -194,8 +194,8 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
             $table->addColumn('version', Types::STRING, ['notnull' => true, 'length' => 255, 'default' => '0.0.1']);
             $table->addColumn('synchronization_id', Types::STRING, ['notnull' => true, 'length' => 255]);
             // Source
-            $table->addColumn('source_id', Types::STRING, ['notnull' => false, 'length' => 255]);
-            $table->addColumn('source_hash', Types::STRING, ['notnull' => false, 'length' => 255]);
+            $table->addColumn('origin_id', Types::STRING, ['notnull' => false, 'length' => 255]);
```

Comment:
We're not updating migrations that have already been merged to main. This will have to be fixed otherwise.